proposals/0XXX-deprecate-rent-exempt-threshold.md

```diff
@@ -0,0 +1,66 @@
+---
+simd: '0XXX'
+title: Deprecate Rent Exempt Threshold
+authors:
+category: Standard
+type: Core
+status: Review
+created: 2024-11-14
+feature:
+supersedes:
+superseded-by:
+extends:
+---
+
+## Summary
+
+This proposal aims to eliminate the use floating-point operations to determine whether an account is rent exempt or not in programs by deprecating the use of the `exempt_threshold` value &mdash; the `exempt_threshold` is currently specified as a `f64`. 
+
+More specifically, it proposes to rely only on the `lamports_per_byte_year` (to be renamed to `lamports_per_byte`), which is a `u64` value, and set the `exemption_threshold` to `1.0`, deprecating its use.
```

Comment:
deprecating its use in SDKs and onchain programs moving forward without affecting the existing API.